deploy_snapshot.sh

```diff
@@ -14,10 +14,10 @@ elif [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
   echo "Skipping snapshot deployment: was pull request."
 elif [ "$TRAVIS_BRANCH" != "$BRANCH" ]; then
   echo "Skipping snapshot deployment: wrong branch. Expected '$BRANCH' but was '$TRAVIS_BRANCH'."
-elif [[ $(./gradlew :PopupBridge:properties | grep version) != *-SNAPSHOT ]]; then
+elif [[ $(./gradlew properties | grep version) != *-SNAPSHOT ]]; then
   echo "Skipping snapshot deployment: not a snapshot version."
 else
   echo "Deploying snapshot..."
-  ./gradlew :PopupBridge:uploadArchives
+  ./gradlew :PopupBridge:publishToSonatype
```

Comment:
Snapshot build scripts will be purged at a later date across all repos. Making the change here for due diligence.